docs/guides/caddy-configuration.md

```diff
@@ -46,17 +50,45 @@ blackhole:80, pi.hole:80, 0.0.0.0:80 {
 
 In this case, I've chosen to also add blackhole and pi.hole as valid names to open the admin page with.
 
-Finally, restart your Caddy server: `sudo service caddy restart`
+### Caddyfile (Caddy version 2)
+
+```
+pi.hole:80{
```

Comment:
Oops - sorry, I just noticed this issue on a second glance, should I make another PR to change it?

`pi.hole:80{` need to be `pi.hole:80 {`, the space is required otherwise the `{` will be considered part of the port and the parsing will fail.

Reply:
Yes please go ahead and PR the change(s) :)